CHANGES.md

```diff
@@ -21,7 +21,7 @@
 * Remove usages of 'failure' crate
   * <https://github.com/georust/geo/pull/388>
 
-## geo-types (unreleased)
+## geo-types 0.4.4
```

Comment:
since there's a breaking change in here, we should bump to 0.5 here, and 0.13.0 in `geo`

Reply:
Oops. Done!